DistributedTransaction/DistributedTransaction.tla

```diff
@@ -102,8 +102,7 @@ ReqMessages ==
   \union  [start_ts : Ts, primary : KEY, type : {"commit"}, commit_ts : Ts]
   \union  [start_ts : Ts, primary : KEY, type : {"resolve_rollbacked"}]
   \union  [start_ts : Ts, primary : KEY, type : {"resolve_committed"}, commit_ts : Ts]
-  \union  [start_ts : Ts, primary : KEY, type : {"check_txn_status_req"}, 
-            rollback_if_not_exist : BOOLEAN, resolving_pessimistic_lock : BOOLEAN]
+  \union  [start_ts : Ts, primary : KEY, type : {"check_txn_status"}, resolving_pessimistic_lock : BOOLEAN]

```

Comment:
Add a comment to explain why `rollback_if_not_exist` is not necessary in TLA+

Reply:
comment added.